### ISSUE TITLE
Aura component retireve and deploy errors using sfdx vscode

### ISSUE THREAD

**Issue by dave-sfdc** [forcedotcom/salesforcedx-vscode#2928]
**### Summary**
I have seen an issue with Visual Studio Code while retrieving and deploying aura components.
The aura components now have new folders for each of the cmp, js files
I created a project with manifest using sfdx extension on VSC
Tried a new VSC project for my local dev org and same behavior, so it is not org related

Basically, the error also complains about extra folders but not sure why does it create them.

### Steps To Reproduce:

sfdx force:source:retrieve --sourcepath
ERROR running force:source:retrieve:  Unexpected file found in package directory:
..\force-app\main\default\aura\Accordion\Accordion.cmp\Accordion.cmp

sfdx force:source:deploy --sourcepath
UnexpectedFileFound: Unexpected file found in package directory:
..\force-app\main\default\aura\Accordion\Accordion.cmp\Accordion.cmp

### Expected result

The aura components should have cmp file, js files etc in a single folder

### Actual result

Multiple folders are created for each file of aura component as shown in screenshots
There are also cmp-meta.xml files created in each folder

also getting error: Error re-indexing workspace: EISDIR: illegal operation on a directory, read
### Additional information

Version: 1.53.0 (user setup)
Commit: 8490d3dde47c57ba65ec40dd192d014fd2113496
Date: 2021-02-03T20:36:38.611Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.18363

SFDX CLI Version : v50.14.0 and v50.16.0

![image](https://user-images.githubusercontent.com/17682618/107286178-38dd5000-6a2e-11eb-9dac-1789dc5e4dd3.png)
![image](https://user-images.githubusercontent.com/17682618/107286228-498dc600-6a2e-11eb-9ce6-699e2d4985a0.png)

**Comment by Rich0635** [comment]
I get this when trying to retrieve from the lwc folder. The aura folder is fine for me. I cannot retrieve any of my LWC components by package.xml, right-click lwc, or using terminal command line.

16:58:53.207 sfdx force:source:retrieve --sourcepath /Users/richardwintle/Documents/VS Workspace/GBC/force-app/main/default/lwc
ERROR running force:source:retrieve:  EISDIR: illegal operation on a directory, read
16:58:59.438 sfdx force:source:retrieve --sourcepath /Users/richardwintle/Documents/VS Workspace/GBC/force-app/main/default/lwc
 ended with exit code 1

**Comment by lcampos** [comment]
This seems similar to issue #2907. @Rich0635 & @dave-sfdc can you try updating to the latest version of the Salesforce CLI ?

**Comment by Rich0635** [comment]
I am already on the latest version 50.16.0
The problem is still there. Interestingly, I created a new project and the problem is not there. Somehow my old project is corrupted or not compatible with CLI new version.

- open new project - it is ok.
- open old project - it is broken, same error.

Creating new project means I lose all my file change history :(

**Comment by dave-sfdc** [comment]
I have tried another folder, another project, and even another org and the same issue is seen. There is more to it as so many people are reporting

**Comment by dave-sfdc** [comment]
Just an update, after trying a couple of things, I moved to sfdx-cli/7.85.1 and that seems to have fixed the issue. Thanks for the assistance @lcampos

**Comment by lcampos** [comment]
@dave-sfdc glad your issue got sorted out by just updating the CLI.

@Rich0635 that version you provided is for the Salesforce extensions for VSCode. On your terminal, run `sfdx version` to get the CLI version and `sfdx update` to download the latest updates. Let us know if that is enough to clear your issues too.

**Comment by Rich0635** [comment]
I updated the sfdx cli

sfdx version is now...
	sfdx-cli/7.86.3 darwin-x64 node-v12.14.1

Unfortunately, I still get the same error.

**Comment by Rich0635** [comment]
Fixed. I found the lwc folder had become corrupted with the component structure. Somehow it had created a folder for each file.
![Screenshot 2021-02-11 at 12 21 00](https://user-images.githubusercontent.com/60474144/107636145-20567b00-6c64-11eb-851f-eb99e1679f6c.png)

I deleted the components in lwc and ran the command line retrieve:
sfdx force:source:retrieve -m LightningComponentBundle

Everything works now!
